dist/wtc-gallery-component.js

```diff
@@ -89,9 +89,9 @@ function (_ElementController) {
 
     if (_this.options.nav) {
       _this.nextBtn = document.createElement('button');
-      _this.nextBtn.innerHTML = 'Next';
+      _this.nextBtn.innerHTML = 'Next <span class="visually-hidden">carousel item.</span>';
```

Comment:
It might be good to make this text a variable or able to be configured in some way. Same for line 94.

Reply:
This is usually replaced with an icon, I feel like this is pretty good and generic, I can't see a use case where we wanna get too specific.

Reply:
I've added this in. I think it's good practice, even if it's only used to pass in different text based on language/region.